index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+<!-- Userform Title line  -->
+  <title>User Form</title>
+  <link rel="stylesheet" href="node_modules/bulma/css/bulma.css">
+  <link rel="stylesheet" href="css/mystyles.css">
+
+  <script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+  <script src="javascript/script.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<!-- Reference to user name and welcome page -->
+  <section class="section">
+    <div class="container">
+      <h1 class="is-size-3"><i class="far fa-user-circle is-size-1"></i> Welcome Back Peppa!</h1>
+    </div>
+  </section>
+
+<!-- Preferences title line  -->
+  <div class="container">
+    <h1 class="is-size-3 has-text-weight-bold">Preferences</h1>
+  </div>
+
+<!-- Month input line -->
+  <section class="section">
+    <div class="container">
```

Comment:
It seems like you were trying to apply multiple classes to one tag and ended up having one div tag for each; there might be a way to combine all of them together in one tag to make it cleaner?